app/webserver/helpers.go

```diff
@@ -89,7 +89,7 @@ func updateCurrentChat(s *store.Session) {
 		c   *textsecureContacts.Contact
 	)
 	if s.IsGroup {
-		gr, err = textsecure.GetGroupById(s.Tel)
+		gr, err = textsecure.GetGroupById(s.UUID)
```

Comment:
I think we should use the `store.GetGroupById` instead of querying the backend: https://pkg.go.dev/github.com/nanu-c/axolotl@v1.0.8/app/store#GetGroupById

Reply:
will try it.

Reply:
Using store.GetGroupById worked.

Unfortunately it introduced the regression that Axolotl leaves conversations that are not groups after I open them.
I'll see what I can do about that.